homepage/static/css/custom.css

```diff
@@ -584,3 +584,28 @@ h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
 section.downloads {
     padding-bottom: 50px;
 }
+.schedule h5 {
+    margin-top: 20px;
+}
+img, embed,
```

Comment:
If it is required only for course site please change it that way that it will not affect all orange website. For example you can add to image where this property is required `full-width` class or something similar and then use `img.full-width video.full-width ...` in css. 
If it is made with purpose to have impact on whole orange website then ignore this comment.